yarn.lock

```diff
@@ -2146,6 +2187,36 @@ alphanum-sort@^1.0.0:
   resolved "https://registry.yarnpkg.com/alphanum-sort/-/alphanum-sort-1.0.2.tgz#97a1119649b211ad33691d9f9f486a8ec9fbe0a3"
   integrity sha1-l6ERlkmyEa0zaR2fn0hqjsn74KM=
 
+analytics-node@^3.4.0-beta.2:
```

Comment:
Need to check that this doesn't bundle anything unrelated we don't want on the client

Reply:
it seems good to me https://github.com/segmentio/analytics-node#readme

Reply:
`analytics-node` is a dependency of `@analytics/segment`, which exports a node and a browser version, depending on `process.browser` being truthy. the node version uses `analytics-node`.

We are importing `@analytics/segment` global to a file imported in _app.js, so I was curious how this would be bundled and quickly made a little test sandbox, once building a mosty blank page, once importing `@analytics/segment`.
The JS bundle for that page was only increased by 0.9kb, indicating that only the small browser-related portion is bundled and not `analytics-node` and its range of dependencies like `axios`.

So yes indeed, everything seems good here :)

Reply:
Thanks! I appreciate the explanation. It's a good way to test it 😅